dashboard/db/migrate/20170118223108_change_best_result_for_contained_levels.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeBestResultForContainedLevels < ActiveRecord::Migration[5.0]
+  def change
+    # Find all existing UserLevels for contained levels, and update their best
+    # result to be CONTAINED_LEVEL_RESULT (101)
+    containers = Level.all.select{|x| !x.contained_levels.empty?}
+    contained_level_ids = containers.map{|x| x.contained_levels[0].id}
+    UserLevel.where(level_id: contained_level_ids).update_all(best_result: 101)
```

Comment:
I assume the answer is that we're not okay with taking down production. Any pointers on how to better implement this? (or examples I can emulate)